pkg/ccl/streamingccl/streamingest/stream_ingestion_frontier_processor.go

```diff
@@ -329,6 +334,9 @@ func (sf *streamIngestionFrontier) maybeUpdateProgress() error {
 		streamProgress := progress.Details.(*jobspb.Progress_StreamIngest).StreamIngest
 		streamProgress.Checkpoint.ResolvedSpans = frontierResolvedSpans
 
+		// Initialize heartbeat time to cutover time to compare with persisted replicated time later.
```

Comment:
nit: i think you can remove this docstring. the reader can infer this by seeing how we use `cutoverTime`